applications/StructuralMechanicsApplication/custom_elements/nodal_concentrated_element.cpp

```diff
@@ -16,65 +16,83 @@
 // Project includes
```

Comment:
why should we have an element that does the two things? (i mean displacements and rotations).

I would say that it is cleaner that we have one element with just displacements and another one with displacements and rotations...

Reply:
We can of course split it, but in case we want to consider both then we will need to duplicate the elements

Reply:
I think we should have
- one that can do both, i.e. disp AND rot
Or
- one for disp and one for rot